libraries/Microsoft.Bot.Builder.Ai.LUIS/Client/LuisService.cs

```diff
@@ -253,8 +258,7 @@ public void ApplyThreshold(LuisResult result)
         async Task<LuisResult> ILuisService.QueryAsync(Uri uri, CancellationToken token)
         {
             string json;
-            using (var client = new HttpClient())
-            using (var response = await client.GetAsync(uri, HttpCompletionOption.ResponseContentRead, token))
+            using (var response = await _httpClient.GetAsync(uri, HttpCompletionOption.ResponseContentRead, token))
             {
```

Comment:
This await needs (I think) the .ConfigureAwait = False. This will mean an actual perf difference here, as it's a network call and rejoining the threads will take a long time. 

If the ConfigureAwait is NOT needed, we should put a comment in as to why.